src/WebJobs.Script/Workers/Rpc/RpcWorkerConstants.cs

```diff
@@ -8,6 +8,8 @@ namespace Microsoft.Azure.WebJobs.Script.Workers.Rpc
     public static class RpcWorkerConstants
     {
         public const string FunctionWorkerRuntimeSettingName = "FUNCTIONS_WORKER_RUNTIME";
+        // semicolon seperated string - list of runtimes to start in placeholder mode
+        public const string FunctionWorkerPlaceholderModeListSettingName = "FUNCTIONS_WORKER_RUNTIME_PLACEHOLDERMODE_LIST";
```

Comment:
Is this being set in all of our environments now?

Reply:
Please see https://github.com/Azure/azure-functions-docker-private/pull/241 

Starting multiple language workers in placeholder mode only applies to Linux Consumption - as we need to optimize existing pool due to capacity restrictions. On Windows - only a single language worker channel is started in placeholder mode - this is controlled via `FUNCTIONS_WORKER_RUNTIME` that is set on the placeholder template sites in Antares.

Do let me know if you have any other concerns.

Reply:
nit: can we seperate `PLACEHOLDERMODE` into `PLACEHOLDER_MODE` - feels and looks cleaner if keep words seperated :)